name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/DeutscheBankPDFExtractor.java

```diff
@@ -442,15 +445,14 @@ private void addAccountStatementTransaction()
                             if (t.getCurrencyCode() != null && t.getAmount() == 0)
                                 item.setFailureMessage(Messages.MsgErrorTransactionTypeNotSupported);
 
+                            if (Boolean.valueOf(type.getCurrentContext().getBoolean("skipTransaction")))
```

Comment:
#getBoolean already returns a boolean. No need to convert it to a boolean via "Boolean#valueOf"

Reply:
i see it... i missing coffee in the morning. :see_no_evil: